### ISSUE TITLE
[19w05a] Typing in the search box crashes the game

### ISSUE THREAD

**Issue by Shnupbups** [shedaniel/RoughlyEnoughItems#31]
Attempting to type anything with the search box selected crashes the game with a stack overflow.

Crash log: [Pastebin](https://pastebin.com/6GiBz8Dg)
(Due to the nature of stack overflows, it's quite... repetitive)

Minecraft Version: 19w05a
Fabric Version: 19w05a.7
Fabric Loader Version: 0.3.5.106
REI Version: 2.2.0.49
Mod List: Found in the crash log

**Comment by Shnupbups** [comment]
Updated to a newer version of Fabric. Still get the crash, but got a more insightful crash log this time.

[Pastebin](https://pastebin.com/6bJEdiX1)

It seems to be an issue with the Torch Item trying to use the Wall Torch Block's translation key, but the Wall Torch Block is trying to use the Torch Item's translation key, resulting in the stack overflow.

So not entirely sure if the crash is fixable on your end?

Only thing I've changed is the mod list, slightly, and updated Fabric to 19w05a.10.
